services/brig/deb/opt/brig/templates/user/de/email/team-activation.html

```diff
@@ -41,10 +41,12 @@
 <tr>
     <td width="500" colspan="2" style="padding-top: 48px;padding-bottom: 0px;">
     <p style="font-size: 32px; font-weight: 300; text-align: center; line-height: 1.3;">
-    Dein neues Wire-Benutzerkonto im '${team}' Team
+    Dein neues Wire Konto
```

Comment:
We use `Wire-Benutzerkonto` everywhere else, please ensure the use is consistent across all emails (either `Wire Konto` or `Wire-Benutzerkonto`. )

Reply:
Thanks for the suggestions. At this point in time, I would like to fix only broken links or major grammar errors. Our copywriter will review the copy of all emails this week to ensure consistency, so expect a better copy/translation to come in this or next week.